resources/theme.json

```diff
@@ -33,6 +34,10 @@
       },
       "Table": {
         "headerBorderRadius": 0
+      },
+      "Layout": {
+        "lightSiderBg": "#F4F7FD",
+        "siderBg": "#2A2C30"
       }
```

Comment:
We need to set the dark `siderBg` and `lightSiderBg` to support the feature of forcing the theme mode of `Sider`.
Also you can change and suggest color value for dark `siderBg`. @agatha197